packages/core/src/plugins/events.ts

```diff
@@ -22,10 +20,10 @@ export default function (_window = globalThisOrWindow()): Types.Plugin {
           return
         }
 
-        // todo: send browser info
-        client.logEvent({
-          level,
-          args
+        client.event('log', {
+          severity: level,
+          message: args[0],
+          args: args.slice(1)
```

Comment:
I could go with `args: args` here, but I thought that usually console logs have a string message as the first argument along with metadata/context as the following arguments (optionally).
@joshuap @stympy Let me know if you have any other thoughts here.

Reply:
How about going with this if args[0] is a string but go with args: args if args[0] is an object? I agree that most of the time `log` is called with a string as arg 0, but there are cases when `log` is called with just an object.

Reply:
Good point, done!